dll/win32/msutb/lang/pt-PT.rc

```diff
@@ -0,0 +1,79 @@
+/*
+ * PROJECT:     ReactOS msutb.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2023-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&OK"
+    IDS_CANCEL           "&Cancelar"
+    IDS_ABORT            "&Abortar"
+    IDS_RETRY            "&Retroceder"
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_SETTINGS         "Definições..."
+    IDS_RESTORELANGBAR   "Restaurar a barra de idiomas"
+    IDS_LANGUAGE         "Linguagem"
+    IDS_LANGUAGEBUTTON   "Botão Linguagem"
+    IDS_TRANSPARENCY     "Transparência"
+    IDS_TEXTLABELS       "Texto da etiqueta"
+    IDS_MINIMIZE         "Minimizar"
+    IDS_CLOSELANGBAR     "Fechar a barra de idiomas"
+    IDS_EXTRAICONS       "Ícones adicionais na barra de tarefas"
+    IDS_LANGUAGEBAR      "A barra de idiomas"
+    IDS_RESTORELANGBAR2  "Restaurar a barra de idiomas"
+    IDS_MENUWND          "Menu Window"
+    IDS_VERTICAL         "Vertical"
+    IDS_ADJUSTLANGBAND   "Ajuste a posição da banda de idioma"
+    IDS_LEFTCLICK        "Clique esquerdo"
+END
+
+IDD_CLOSELANGBARNOBAND DIALOGEX 0, 0, 200, 105
+CAPTION "Barra de idiomas"
+STYLE WS_POPUPWINDOW | WS_CAPTION | DS_MODALFRAME
+FONT 8, "MS Shell Dlg"
+{
+    LTEXT "Fechou a barra de idiomas.\r\n\r\nPara utilizar a barra de idiomas posteriormente, marque ""Mostrar a barra de idiomas na área de trabalho"" na caixa de diálogo Configurações da barra de idiomas, no painel de controle dos Serviços de Texto.", -1, 5, 5, 190, 60
+    AUTOCHECKBOX "&Não mostrar esta mensagem novamente.", IDC_CLOSELANGBAR_CHECK, 5, 70, 145, 10
+    DEFPUSHBUTTON "&OK", IDOK, 80, 85, 55, 15
+    PUSHBUTTON "&Cancelar", IDCANCEL, 140, 85, 55, 15
+}
+
+IDD_CLOSELANGBAR DIALOGEX 0, 0, 200, 105
+CAPTION "Barra de idiomas"
+STYLE WS_POPUPWINDOW | WS_CAPTION | DS_MODALFRAME
+FONT 8, "MS Shell Dlg"
+{
+    LTEXT "Fechou a barra de idiomas.\r\n\r\nPara utilizar a barra de idiomas posteriormente, marque ""Mostrar a barra de idiomas na área de trabalho"" na caixa de diálogo Configurações da barra de idiomas, no painel de controle Opções regionais e de idioma.", -1, 5, 5, 190, 60
+    AUTOCHECKBOX "&Não mostrar esta mensagem novamente.", IDC_CLOSELANGBAR_CHECK, 5, 70, 145, 10
+    DEFPUSHBUTTON "&OK", IDOK, 80, 85, 55, 15
+    PUSHBUTTON "&Cancelar", IDCANCEL, 140, 85, 55, 15
```

Comment:
Please prove with a screenshot from portugese 2k3sp2 that Windows does use an accelerator here for OK and Cancelar. You did that in multiple dialogs, and I do somehow have my doubts that it is correct, based on findings in other translations.

Reply:
You're right. There are no accelerators on the Ok and Cancel buttons, at least in WinXP. Thanks. Done.